src/settings_table.cpp

```diff
@@ -443,6 +443,33 @@ static bool CheckMaxHeightLevel(int32 &new_value)
 	return true;
 }
 
+static bool CheckModifier(int32 &new_value, ModifierKey other1, ModifierKey other2)
```

Comment:
Technically and practically you do not need the `&` here anymore, since you're not going to change the value.
I'd maybe even use `ModifierKey key` here as parameter, and do the cast in the functions below.